src/hotspot/share/runtime/safefetch.inline.hpp

```diff
@@ -32,13 +32,17 @@
 // to be valid. If the load causes a fault, the error value is returned.
 inline int SafeFetch32(int* adr, int errValue) {
   assert(StubRoutines::SafeFetch32_stub(), "stub not yet generated");
-  MACOS_AARCH64_ONLY(ThreadWXEnable wx(WXExec, Thread::current()));
+  Thread* thread = Thread::current_or_null_safe();
```

Comment:
Sorry but this should be MACOS_AARCH64 only. All three lines need to be ifdef'd if you are going to include the assertion.

Thanks,
David

Reply:
Right, thanks! Fixed in https://github.com/openjdk/jdk/pull/2200/commits/3d0f4d2342adc867eaf762fa83a9c3035d6439bd